src/Compilers/CSharp/Portable/CSharpCodeAnalysis.csproj

```diff
@@ -348,7 +348,7 @@
     <Compile Include="FlowAnalysis\AlwaysAssignedWalker.cs" />
     <Compile Include="FlowAnalysis\ControlFlowAnalysis.cs" />
     <Compile Include="FlowAnalysis\ControlFlowPass.cs" />
-    <Compile Include="FlowAnalysis\DataFlowAnalysis.cs" />
+    <Compile Include="FlowAnalysis\CSharpDataFlowAnalysis.cs" />
```

Comment:
Yes; I changed the name of the file to reflect the name of the class it contains.